From: Yao Wei <yao.wei@canonical.com>
Date: Tue, 27 Jul 2021 14:05:28 +0800
Subject: [PATCH] Use SKU to identify Dell clamshell models for accelerometer
 properties

As per request from Dell, we need to use sku property of modalias that
is patched into linux 5.13 for model identification:

  https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=e26f023e01ef26b4138bc1099af309bdc4523d23
---
(edited by ddstreet to leave existing lines instead of replacing them)

 hwdb.d/60-sensor.hwdb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -288,9 +288,11 @@ sensor:modalias:platform:HID-SENSOR-2000
  ACCEL_LOCATION=base
 
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:pnLatitude9520:*:ct10:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0A3E:*
  ACCEL_LOCATION=base
 
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:pnLatitude7420:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B0B:*
  ACCEL_LOCATION=base
 
 # Dell Venue 8 Pro 3845
